### PR TITLE
Add new files (profile manager, whatsnew63), update devedition

### PR DESCRIPTION
**Pull request by peiying2**
* firefox/products/developer-quantum.lang

### CODE REVIEW

**Comment by flodolo** [comment]
This looks good, but there's already another PR tagged l10n.

File: en-US/firefox/profile-per-install.lang [hunk 1]
```diff
@@ -0,0 +1,68 @@
+## NOTE: Demo pages at https://bedrock-demo-craigcook.oregon-b.moz.works/firefox/profile-downgrade/ and https://bedrock-demo-craigcook.oregon-b.moz.works/firefox/profile-migrate/
+## NOTE: On new changes in profile management to users with multiple installs
+## NOTE: See details here https://github.com/mozilla/bedrock/pull/6168
+## URL: https://bedrock-demo-craigcook.oregon-b.moz.works/%LOCALE%/firefox/profile-downgrade/ and https://bedrock-demo-craigcook.oregon-b.moz.works/%LOCALE%/firefox/profile-migrate/
```
Comment (flodolo) [inline en-US/firefox/profile-per-install.lang]: You can only have on element in this

**Comment by TonnesM** [comment]
"... the guesswork out of fine-tuning your shadow-outside and clip-path shapes ..."

I wonder if "shadow-outside" is correct. Shouldn’t that be "shape-outside" instead?

**Comment by flodolo** [comment]
Sorry, I really have no clue (just passing papers here…). 

Can you try asking here to who wrote the page?
https://github.com/mozilla/bedrock/pull/6085

**Pull request merged**